extension/src/test/suite/experiments/util.ts

```diff
@@ -218,19 +256,17 @@ export const buildExperimentsData = (
 
 export const stubWorkspaceExperimentsGetters = (
```

Comment:
[F] This function needs a rewrite now (should use `buildExperiments`) but doing that in this PR will just add even more noise. I'll branch off this and make the change.